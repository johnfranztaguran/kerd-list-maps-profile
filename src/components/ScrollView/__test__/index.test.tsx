import React from 'react'
import { Text } from 'react-native'
import { render } from '@testing-library/react-native'
import { ScrollView } from '../index'

describe('Component: ScrollView', () => {
  it('Should render a <ScrollView />', () => {
    const { baseElement } = render(
      <ScrollView>
        <Text>Child Element</Text>
      </ScrollView>
    )
    expect(baseElement).toMatchSnapshot()
  })
})
