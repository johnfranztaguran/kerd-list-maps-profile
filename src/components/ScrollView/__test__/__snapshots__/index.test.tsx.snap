// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: ScrollView Should render a <ScrollView /> 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <ScrollView
      accessibilityLabel="scrollview--screen"
      accessible={true}
      scrollEventThrottle={200}
      testID="scrollview--screen"
    >
      <Text>
        Child Element
      </Text>
    </ScrollView>
  </View>
</View>
`;
